scipy/interpolate/fitpack2.py

```diff
@@ -23,7 +23,7 @@
 
 import warnings
 
-from numpy import zeros, concatenate, alltrue, ravel, all, diff, array, ones
+from numpy import zeros, concatenate, ravel, all, diff, array, ones
```

Comment:
If we're changing these, I'd prefer to use the `np.` prefixed versions instead.

That is, let's replace `alltrue` with `np.all` instead of importing `all` (and shadowing the python builtin).

Reply:
That's a good point, I'll change that.